ci_framework/roles/override_op_image/README.md

```diff
@@ -0,0 +1,14 @@
+# override_op_image
+A role used to override the operator images used in the deployment. A `yaml` file is passed to the ansible playbook (usually the file generated by `env_op_images` role from a previous deployment) containing the images with tags to be used for the current deployment
+
+## Parameters
+* `cifmw_override_op_image_dir`: (String) The directory where the updated `OpenstackControlPlane` CR yaml used for the deployment is stored. Defaults to `{{ cifmw_basedir }}/artifacts/manifests/openstack`
+
+## Examples
+```YAML
+- name: Override openstack service images
+  when:
+  - cifmw_openstack_svc_image is defined
```

Comment:
apparently pySpelling is crying due to that "svc" mention. Care to add it to the dictionary, as described in the doc here: https://ci-framework.readthedocs.io/en/latest/internals/02_testing.html#spellchecking